WATOLINK/bci-gaming
# Alexis menu styling

## ClickUp ticket link

[Design - Menu Modals](https://app.clickup.com/t/860qed169)



## Implementation description
Created buttons and modals to fit theming of the western game and pixel art



## Steps to test
1. Play the game to ensure the menu is in frame. 
2. Click on the info, settings, and bci menu buttons to see the modal
3. click play to ensure it switches to the empty game frame



## What should reviewers focus on?
Opinions on the styling of the buttons/ modals


## Checklist
